src/client/gameui.cpp

```diff
@@ -334,3 +334,15 @@ void GameUI::deleteFormspec()
 
 	m_formname.clear();
 }
+void GameUI::Clear()
+{
+	
+	if(m_guitext_chat)
```

Comment:
1. use camelCase for the function name -> `clear`
2. Space after `if`
3. Opening curly brace into the same line or omit

Please have a look at https://dev.minetest.net/Code_style_guidelines

Reply:
Okay, I'm sorry.